tripleblindmarket/covid-safe-paths
# [RC] add release-candidate and master to branches for workflow testing

## Description

Adds GitHub workflow testing to the `release-candidate` and `master` branches.  Adding master as I'm assuming soon we'll be merging over to the `master` branch from `release-candidate`

## How to test

On the PR for this make sure the test ran. (I will start with `release-candidate` first)
